Subject: [PATCH] Enable IMGUI freetype

---
 imconfig.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imconfig.h b/imconfig.h
index e3dc27f6..5dcdf61f 100644
--- a/imconfig.h
+++ b/imconfig.h
@@ -73,7 +73,7 @@
 //---- Use FreeType to build and rasterize the font atlas (instead of stb_truetype which is embedded by default in Dear ImGui)
 // Requires FreeType headers to be available in the include path. Requires program to be compiled with 'misc/freetype/imgui_freetype.cpp' (in this repository) + the FreeType library (not provided).
 // On Windows you may use vcpkg with 'vcpkg install freetype --triplet=x64-windows' + 'vcpkg integrate install'.
-//#define IMGUI_ENABLE_FREETYPE
+#define IMGUI_ENABLE_FREETYPE
 
 //---- Use stb_truetype to build and rasterize the font atlas (default)
 // The only purpose of this define is if you want force compilation of the stb_truetype backend ALONG with the FreeType backend.
-- 
2.37.1.windows.1

